src/resources/postcss/tribe-common-admin/settings/_tabbed-nav.pcss

```diff
@@ -193,7 +195,11 @@
 			}
 
 			.tec-nav__subnav {
+				border-inline: 1px solid var(--tec-admin-core-grey-10);
+				border-top: 1px solid var(--tec-admin-core-grey-10);
 				left: var(--tec-spacer-7);
+				/* This calc is to remove each of the 1px borders */
+				max-width: calc(var(--tec-width-settings-form) - 2px);
```

Comment:
```suggestion
				box-sizing: border-box;
				max-width: var(--tec-width-settings-form);
```

Reply:
(and remove the comment)

Reply:
We actually need this to be done via `calc()`. I think the `absolute` positioning of this element is causing the border to be added to the element width, and if you look closely you can see the edges don't align.

Here's a screenshot with the suggested change in place:

<img width="961" alt="Screenshot 2024-08-21 at 21 20 06" src="https://github.com/user-attachments/assets/c9f64b93-cb0d-4e43-a5ba-d49f43c76d24">

The 2px extra length is visible on the right side.

Reply:
🤦🏻‍♂️ Nevermind, I overlooked the addition of the `box-sizing` style.